src/languages/index.js

```diff
@@ -50,24 +48,34 @@ function getTranslationByKey(translations, key, lang) {
     : `Invalid translation format for key '${key}' in language '${lang}'.`;
 }
 
-const interpolateText = (text, props = {}) => {
+const interpolateText = (text, params = {}) => {
   let result = text;
-
-  Object.entries(props).forEach(([key, value]) => {
+  Object.entries(params).forEach(([key, value]) => {
     result = result.replaceAll(`{{${key}}}`, value);
   });
-
   return result;
 };
 
-function translateLanguage(key, params = {}) {
-  const translations = loadTranslations(botLanguage);
-  if (!translations || !Object.values(translations).length) {
-    return `Translations not available for language '${botLanguage}'.`;
+function translateLanguage(key, params = {}, language = botLanguage) {
+  const translations = translationsCache[language] || {};
+  if (!Object.keys(translations).length) {
+    return `Translations not available for language '${language}'.`;
   }
-
-  const translation = getTranslationByKey(translations, key, botLanguage);
+  const translation = getTranslationByKey(translations, key, language);
   return interpolateText(translation, params);
 }
 
-module.exports = { translateLanguage };
+const translateCommand = (command) => {
```

Comment:
translateCommand could be a bit confusing.

For example I recently read that receive the param: `command` and thought it received an `order`, like on CQRS, instead of receive the key of a translation

Reply:
Would be better to name it by what it does and not by what it is being used for.

What it does is returning the translations of a key in all their languages